build.sbt

```diff
@@ -4,21 +4,23 @@ scalaModuleSettings
 
 name                       := "scala-partest"
 
-version                    := "1.0.11-SNAPSHOT"
+version                    := "1.0.12-SNAPSHOT"
 
 scalaVersion               := crossScalaVersions.value.head
 
 crossScalaVersions         := {
   val java = System.getProperty("java.version")
   if (java.startsWith("1.6."))
-    Seq("2.11.6", "2.12.0-M1")
+    Seq("2.11.7", "2.12.0-M1")
   else if (java.startsWith("1.8."))
-    Seq("2.12.0-M2")
+    Seq("2.12.0-M2", "2.12.0-M3")
   else
     sys.error(s"don't know what Scala versions to build on $java")
 }
 
-scalaXmlVersion            := "1.0.4"
+scalaXmlVersion            := {
+  if(scalaVersion.value == "2.12.0-M3") "1.0.5" else "1.0.4"
```

Comment:
When we do M4 (and it's not binary compatible), we need to update scala-partest anyway (and scala-xml may well be an even newer version then), so I don't think it really makes a difference which way you write that check.